ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-cst-materialized.json

```diff
@@ -68,6 +68,21 @@
           {"row":{"columns":[2,"a2"]}}
         ]}
       ]
+    },
+    {
+      "name": "disallowed column in pull query select clause- ROWPARTITION",
```

Comment:
nit: can we have a consistent naming scheme for all the newly added RQTTs scattered across the different pull query RQTT files? For example, "should not allow pull query with ROWPARTITION in SELECT clause" for this one, and similarly for the others below.